generators/app/index.js

```diff
@@ -71,9 +77,19 @@ module.exports = class extends Generator {
     this.log('DbName', this.questions.DbName);
     this.log('auth', this.questions.auth);
     this.log('secretKey', this.questions.secretKey);
+    this.log('packageManager', this.questions.packageManager);
   }
 
   writing() {
+    const install = (packages, options) => {
+      if (this.options.packageManager === 'npm') {
+        this.npmInstall(packages, options);
```

Comment:
return after this; exit early and fast.

Reply:
Cool catch, origionally had bower as a 3rd option, but removed it